### PR TITLE
scripts: typo fixes

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -I ../mantid-word-whitelist.txt -L mye,amin scripts/`

### CODE REVIEW

File: scripts/Calibration/tube.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def calibrate(ws, tubeSet, knownPositions, funcForm, **kwargs):
     Although the examples consider only Gaussian peaks, it is possible to
     change the function factors to edges by passing the index of the
     known_position through the **funcForm**. Hence, considering three special
-    points, where there are one gaussian peak and thow edges, the calibrate
+    points, where there are one gaussian peak and two edges, the calibrate
```
Comment (luzpaz) [inline scripts/Calibration/tube.py]: please review this fix

File: scripts/HFIR_4Circle_Reduction/peakprocesshelper.py [hunk 2]
```diff
@@ -306,7 +306,7 @@ def get_weighted_peak_centres(self):
         """
         # get PeaksWorkspace
         if AnalysisDataService.doesExist(self._myPeakWorkspaceName) is False:
-            raise RuntimeError('PeaksWorkspace %s does ot exit.' % self._myPeakWorkspaceName)
+            raise RuntimeError('PeaksWorkspace %s does not exist.' % self._myPeakWorkspaceName)
```
Comment (luzpaz) [inline scripts/HFIR_4Circle_Reduction/peakprocesshelper.py]: please review

File: scripts/HFIR_4Circle_Reduction/reduce4circleControl.py [hunk 3]
```diff
@@ -2198,7 +2198,7 @@ def set_detector_center(self, exp_number, center_row, center_col, default=False)
             'Center row number {0} of type {1} must either None or non-negative integer.' \
             ''.format(center_row, type(center_row))
         assert center_col is None or (isinstance(center_col, int) and center_col >= 0), \
-            'Center column number {0} of type {1} must be either Noe or non-negative integer.' \
+            'Center column number {0} of type {1} must be either None or non-negative integer.' \
```
Comment (luzpaz) [inline scripts/HFIR_4Circle_Reduction/reduce4circleControl.py]: please review

File: scripts/Inelastic/Direct/DirectEnergyConversion.py [hunk 4]
```diff
@@ -1922,7 +1922,7 @@ def get_failed_spectra_list_from_masks(masked_wksp,prop_man):
     try:
         masked_wksp.name()
 #pylint: disable=broad-except
-    except Exeption:
+    except Exception:
```
Comment (luzpaz) [inline scripts/Inelastic/Direct/DirectEnergyConversion.py]: please review source change

File: scripts/Interface/ui/sans_isis/sans_data_processor_window.ui [hunk 5]
```diff
@@ -415,7 +415,7 @@ QGroupBox::title {
                         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The save locations for the output of batch reductions are: Memory (RAM), File (Hard disk), Both (RAM and Hard disk).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                        </property>
                        <property name="text">
-                        <string>Memor&amp;y</string>
+                        <string>Memory&amp;y</string>
```
Comment (luzpaz) [inline scripts/Interface/ui/sans_isis/sans_data_processor_window.ui]: I will revert this commit.

File: scripts/PyChop/ISISDisk.py [hunk 6]
```diff
@@ -210,7 +210,7 @@ def setEi(self, Ei):
 
     def getEi(self):
         """
-        Returns the focussed incident energy in meV
+        Returns the focused incident energy in meV
```
Comment (luzpaz) [inline scripts/PyChop/ISISDisk.py]: will revert this commit

File: scripts/PyChop/MulpyRep.py [hunk 7]
```diff
@@ -164,7 +164,7 @@ def calcChopTimes(efocus, freq, instrumentpars, chop2Phase=5):
     """
     A method to calculate the various possible incident energies with a given chopper setup on LET.
     The window of energy transfers plotted is 85% by default.
-    efocus: The incident enrgy that all choppers are focussed on
+    efocus: The incident enrgy that all choppers are focused on
```
Comment (luzpaz) [inline scripts/PyChop/MulpyRep.py]: will be reverted

File: scripts/SCD_Reduction/ReduceSCD.config [hunk 8]
```diff
@@ -194,18 +194,17 @@ run_nums  8512:8523
 run_nums  8525:8551
 
 #
-# Specify the slurm partion, or None to use local processes.  The parameter
-# max_processes controls the maximum number of processes that will be run 
+# Specify the slurm partition, or None to use local processes.  The parameter
```
Comment (luzpaz) [inline scripts/SCD_Reduction/ReduceSCD.config]: is this partition or portion?

File: scripts/SCD_Reduction/TOPAZ.config [hunk 9]
```diff
@@ -161,17 +161,17 @@ reduce_one_run_script  ReduceSCD_OneRun.py
 #run_nums  8525:8551
 
 #
-# Specify the slurm partion, or None to use local processes.  The parameter
-# max_processes controls the maximum number of processes that will be run 
+# Specify the slurm partition, or None to use local processes.  The parameter
```
Comment (luzpaz) [inline scripts/SCD_Reduction/TOPAZ.config]: partition or portion ?

File: scripts/test/SANS/user_file/state_director_test.py [hunk 10]
```diff
@@ -197,7 +197,7 @@ def test_state_can_be_created_from_valid_user_file_with_data_information(self):
         if os.path.exists(user_file_path):
             os.remove(user_file_path)
 
-    def test_stat_can_be_crated_from_valid_user_file_and_later_on_reset(self):
+    def test_stat_can_be_created_from_valid_user_file_and_later_on_reset(self):
```
Comment (luzpaz) [inline scripts/test/SANS/user_file/state_director_test.py]: please review

**Review by luzpaz — COMMENTED** [review]
I've added some comments where I wasn't sure about the validity of some of the fixes. Please also know that I will revert the ones mentioned above when I get the green light on all the other questions.

File: scripts/PyChop/Instruments.py [hunk 11]
```diff
@@ -243,7 +243,7 @@ def getFrequency(self):
         return self.frequency
 
     def setEi(self, Ei):
-        """Sets the (focussed) incident energy"""
+        """Sets the (focused) incident energy"""
```
Comment (AndreiSavici) [inline scripts/PyChop/Instruments.py]: British spelling

File: scripts/PyChop/MulpyRep.py (hunk 7 above)
Comment (AndreiSavici) [reply scripts/PyChop/MulpyRep.py]: British spelling

File: scripts/SCD_Reduction/TOPAZ.config (hunk 9 above)
Comment (AndreiSavici) [reply scripts/SCD_Reduction/TOPAZ.config]: "partition" is the correct choice

File: scripts/SCD_Reduction/ReduceSCD.config (hunk 8 above)
Comment (AndreiSavici) [reply scripts/SCD_Reduction/ReduceSCD.config]: "partition" is the correct choice

File: scripts/PyChop/ISISDisk.py (hunk 6 above)
Comment (AndreiSavici) [reply scripts/PyChop/ISISDisk.py]: British spelling

File: scripts/Inelastic/Direct/DirectEnergyConversion.py (hunk 4 above)
Comment (AndreiSavici) [reply scripts/Inelastic/Direct/DirectEnergyConversion.py]: Changes are OK

File: scripts/HFIR_4Circle_Reduction/reduce4circleControl.py (hunk 3 above)
Comment (AndreiSavici) [reply scripts/HFIR_4Circle_Reduction/reduce4circleControl.py]: Changes are OK

File: scripts/HFIR_4Circle_Reduction/peakprocesshelper.py (hunk 2 above)
Comment (AndreiSavici) [reply scripts/HFIR_4Circle_Reduction/peakprocesshelper.py]: Changes are OK

File: scripts/Calibration/tube.py (hunk 1 above)
Comment (AndreiSavici) [reply scripts/Calibration/tube.py]: "two" is correct

**Review by AndreiSavici — CHANGES_REQUESTED** [review]
Awesome PR. Revert the changes for "focused" instead of "focussed" and "Memory&amp;y"

**Comment by luzpaz** [comment]
All incorrectly fixed commits have been reverted.

**Review by AndreiSavici — APPROVED** [review]

**Pull request merged**